icepyx/core/query.py

```diff
@@ -303,6 +300,25 @@ def product_version(self):
         """
         return self._version
 
+    @property
+    def temporal(self):
+        """
+        Return the Temporal object containing date/time range information for the query object.
+
+        See Also
+        --------
+        icepyx.core.Temporal
+
+        Examples
+        --------
+        >>> reg_a = Query('ATL06',[-55, 68, -48, 71],['2019-02-20','2019-02-28'])
+        >>> print(reg_a.temporal)
+        Start date and time: 2019-02-20 00:00:00
+        End date and time: 2019-02-28 23:59:59
+
+        """
+        return self._temporal
+
```

Comment:
Should we add a `if not hasattr...` statement here to let the user know if temporal parameters weren't set (e.g. as in `start_time`)?

Reply:
This is a good thing to think about.

If it helps, the current state of the code does not have a start_time/end_time variable, nor does it seem to store any information if these were not passed in (a warning is just thrown to the user).